auth-web/vue.config.js

```diff
@@ -3,6 +3,19 @@ module.exports = {
     devtool: 'source-map'
   },
   devServer: {
-    proxy: 'https://auth-api-dev.pathfinder.gov.bc.ca/'
+    proxy: {
+      '/auth/api/*': {
+        target: 'https://auth-api-dev.pathfinder.gov.bc.ca/api/v1', // if your local server is running , use that here
+        pathRewrite: {
+          '/auth/api/': ''
+        }
+      },
+      '/pay/api/*': { // TODO I havent tested this..but hopefully it will work..
```

Comment:
Is the TODO still valid? :)

Reply:
kind of :-) 
This is only for local development and i havent tested payment in local. Hopefully it will work or first person to work in payment on local will have to figure it out...